client/src/main/java/com/vesoft/nebula/client/graph/net/Session.java

```diff
@@ -100,6 +100,99 @@ public synchronized ResultSet execute(String stmt) throws IOErrorException {
         }
     }
 
+    /**
+     * Execute the nGql sentence.
+     * Date and Datetime will be returned in UTC
+     *             JSON struct:
+     *             {
+     *               "results":[
+     *                 {
+     *                   "columns":[],
+     *                   "data":[
+     *                     {
+     *                       "row":row-data,
+     *                       "meta":metadata]
```

Comment:
breaken format?

Reply:
nice catch, hard copied from Yichen's pr, might be his is also broken